hack/dev/hypershft-install.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -euo pipefail
+
+REGION="${HYPERSHIFT_REGION:-us-east-1}"
+BUCKET_NAME="${HYPERSHIFT_BUCKET_NAME:-hypershift-ci-oidc}"
+AWSCREDS="${HYPERSHIFT_AWS_CREDS:-$HOME/.aws/credentials}"
+
+# Install hypershift
```

Comment:
do we want to setup external dns https://hypershift-docs.netlify.app/how-to/aws/external-dns/
hypershift install --external-dns-provider=aws --external-dns-credentials=route53-aws-creds --external-dns-domain-filter=service-provider-domain.com ...?

Reply:
that would definitely be useful for AWS, but not for AKS. Should we have different scripts for the different environments where we may want to install? (we could start by renaming this one with `-aws`) and install external-dns, wdyt?

Reply:
One entrypoint that everyone uses, which knows how to validate the expected input based on the target you're deploying to? :D

Reply:
fwiw we do use external DNS atm in aks as well. But yes let's have a way to install the opinions per platform.